Modules/TOF/src/TrendingRate.cxx

```diff
@@ -93,11 +93,16 @@ void TrendingRate::computeTOFRates(TH2F* h, std::vector<int>& bcInt, std::vector
   mPreviousPlot->Reset();
   mPreviousPlot->Add(h);
 
+  if (nb == 0) { // threshold too high? return since hback was not created
```

Comment:
Ciao @noferini indeed, now I get the case. This is how you suggested

Reply:
fine